pkg/kubelet/dockertools/exec.go

```diff
@@ -110,6 +127,24 @@ func (*NativeExecHandler) ExecInContainer(client DockerInterface, container *doc
 	if err != nil {
 		return fmt.Errorf("failed to exec in container - Exec setup failed - %v", err)
 	}
+
+	if resize != nil {
```

Comment:
Yes and no. I have to run a goroutine to read from the `resize` channel, but what I do with the sizes varies by exec handler (nsenter vs Docker vs rkt).